docs/user_interface.rst

```diff
@@ -25,284 +24,412 @@ If you do processing or acquisition which produces new data, its associated disp
 
 See :ref:`display-panels`.
 
-.. _Data Panel:
+Utility Panels
+==============
 
-Data Panel
-----------
-The Data panel (:menuselection:`Window --> Data Panel`) lists the display items in your project.
+Utility panels are individual windows within the software that serve different functions. While the display panels display data items and allow for customization, the utility panels are where the customization actually happens. For example, a display panel can display image data in different colors, but in order to choose which color to display, the option must be selected in the Inspector utility panel. Each utility panel serves to change or edit a given parameter about the selected data.
 
-You can drag items from the data panel to display panels in the workspace.
+Having individual utility panels for similar parameters means that the workspace can be fully customized for the user's purpose. There are several ways to customize the utility panel layout:
 
-You can delete display items (and associated data items if they become orphaned) by selecting them in the data panel and pressing the delete key.
+* Choose to show or hide any utility panel by using the [Window] menu.
 
-You can filter (by text) what items appear in the data panel by typing into the :guilabel:`Filter` field at the bottom of the data panel. The filter works on the title and the size/data type fields.
+* Quickly close a utility panel by clicking the x button in the corner of its title bar.
 
-.. _Output Panel:
+* Click and drag a utility panel by its title bar to rearrange the layout of the shown panels.
 
-Output
-------
-The Output panel (:menuselection:`Window --> Output`) shows text output while running the application.
+* Layer utility panels by dropping one utility panel onto another. If two or more panels are layered, selection buttons will appear at the bottom of the panel to switch between them.
 
-Additional debugging information may be available using debugger consoles if launched from the console.
+* Pop utility panels out into a separate window by dragging the panel away from the main window by the title bar.
 
-.. _Info Panel:
+* Resize a utility panel by clicking and dragging from its edges.
 
-Info
-----
-The Info panel (:menuselection:`Window --> Info`) shows the position of the cursor and the value at the pixel under the cursor if the cursor is over a display item showing a single data item in a display panel. It will also show the intensity level of the data represented at the cursor position in the histogram panel.
+.. image:: graphics/customize_utility_panels.png
+    :width: 396
+    :alt: Customizing Utility Panels
+    :align: center
```

Comment:
You should remove all of these :align: commands. They are used for inline images, not block images. We want most (if not all) images to be block images. We can discuss alignment separately; but that would be controlled by CSS, not the :align: directive.